pr_3/javascript/random_samples/archive_path_overwrite.js

```
@@ -0,0 +1,99 @@
+    unzip.Parse(zipContents).on('entry', function (entry) {
+        var fileName = 'output/path/' + entry.path;
+        // Arbitrary file overwrite
+        // ruleid:zip_path_overwrite2
+        fs.writeFileSync(fileName, entry.contents);
+    });
+});
+// {/fact}
+
+// {fact rule=path-traversal@v1.0 defects=1}
+//admzip
+const fs = require('fs');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'fs'. Consider to delete it or rename it.

'fs' is defined here:


(lint/suspicious/noRedeclare)